packages/components/src/components/Table/TableContainer.tsx

```diff
@@ -0,0 +1,18 @@
+import { ReactNode } from 'react';
+import styled from 'styled-components';
+
+import { Card } from '../Card/Card';
+
+export const Container = styled.div`
+    flex-direction: column;
+`;
+
+interface TableContainerProps {
+    children: ReactNode;
+}
+
+export const TableContainer = ({ children }: TableContainerProps) => (
+    <Card paddingType="none">
+        <Container>{children}</Container>
+    </Card>
```

Comment:
I believe that this component shouldn't be part of DS component but it should occur in its usage.